### ISSUE TITLE
Zero-shot for MSRVTT retrieval results

### ISSUE THREAD

**Issue by vateye** [salesforce/ALPRO#8]
Hi, I have tried to use the provided pretrained checkpoint of ALPRO to finetune on MSRVTT.

Before start training, it would test on the validation set, and it gives me the results:

{'text2video': {'r1': 17.9, 'r5': 40.0, 'r10': 50.9, 'medianR': 10.0, 'meanR': 57.259}, 'video2text': {'r1': 16.0, 'r5': 34.6, 'r10': 46.0, 'medianR': 13.0, 'meanR': 61.448}

I think it does not match the results presented in the paper which achieves R@1 with 24.1, R@5 with 44.7. I am wondering why it would happen?

**Comment by dxli94** [comment]
Hi,

Results in the paper are reported on the test set. 

Thanks.
